src/main/java/Animal/Main.java

```diff
@@ -0,0 +1,40 @@
+package Animal;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class Main {
+  public static void main(String[] args) {
+    // Dog、Cat、Pigのインスタンス化
+    Animal animal1 = new Dog(1, "pochi");
+    Animal animal2 = new Cat(5, "tama");
+    Animal animal3 = new Pig(3, "boo");
+
+    List<Animal> animalList = new ArrayList<>();
+    animalList.add(animal1);
+    animalList.add(animal2);
+    animalList.add(animal3);
+
+    List<Animal> animalListByAge = animalList.stream()
+        .sorted(Comparator.comparingInt(Animal::getAge))
+        .toList();
+
+    // Animalのage順にcryメソッドを実行
+    for (Animal a : animalListByAge) {
```

Comment:
こういう変数も丁寧につけてあげた方が良いですね。
```suggestion
    for (Animal animal: animalListByAge) {
```

Reply:
ご指摘ありがとうございます！対応しました。
[1b1de22](https://github.com/YaraiM/JavaNewChapter9Assignment/pull/1/commits/1b1de220181ba8240775947bbe16947a89301073)